docs/openapi.yml

```diff
@@ -336,7 +336,7 @@ components:
           type: string
           format: date-time
         owner:
-          description: The owner of the namespace.
+          description: The connectionUrl of the namespace.
```

Comment:
You'll want to be careful with global renames. Not sure this is what you intended?

Reply:
Good call - sorry. Will be fixed in next revision.

Reply:
what is the next revision? can it be fixed here?

Reply:
Yup - the next version of this patch I upload will have the fix.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.